src/navigation/approach_post.py

```diff
@@ -48,6 +48,9 @@ def evaluate(self, ud) -> str:
                 in_odom=self.context.use_odom,
             )
             if arrived:
+                self.context.env.arrived_at_post = True
+                self.context.env.last_post_location = self.context.env.current_fid_pos(odom_override=False)
+                print(f"set last post location to {self.context.env.last_post_location}.")
```

Comment:
Up to you but I think we should remove this if it was just for debugging

Reply:
(just the print statement obviously)

Reply:
talked about at the meeting